sycl/doc/syclcompat/README.md

```diff
@@ -217,24 +223,112 @@ auto range = sycl::nd_range<3>{blocksPerGrid * threadsPerBlock,
 syclcompat::launch<vectorAdd>(range, d_A, d_B, d_C, n);
 ```
 
-Note that since `syclcompat::launch` accepts a device function, the kernel lambda is constructed by SYCLcompat behind the scenes. This means that, for example, `sycl::local_accessor`s cannot be declared. Instead, users wishing to use local memory should launch with a `launch_policy` object as described below.
+Note that since `syclcompat::launch` accepts a device function, the kernel
+lambda is constructed by SYCLcompat internally. This means that, for
+example, `sycl::local_accessor`s cannot be declared. Instead, users wishing to
+use local memory should launch with a `launch_policy` object as described below.
 
 #### launch_policy
 
 In addition to the simple `syclcompat::launch` interface described above,
-SYCLcompat provides a `launch_policy` class, and associated `syclcompat::launch` interface which can be used to pass more complex requirements, including:
+SYCLcompat provides a more flexible (`experimental`) interface to `launch` a
+kernel with a given `launch_policy`. By constructing and passing a
+`launch_policy`, users can pass `sycl::ext::oneapi::experimental::properties`
+associated with the kernel or launch, as well as request **local memory** for
+the kernel.
 
- - local memory size
- - kernel_properties
- - launch_properties
+In order to disambiguate the variadic constructor of `launch_policy`, the
```

Comment:
Nit: The reason (disambiguation) reads like implementation detail.

Reply:
You're right, but this is a README rather than a spec document, and I think it's useful info.